packages/datagovmy-ui/src/icons/agency.tsx

```diff
@@ -1331,6 +1331,8 @@ export const AgencyIcon: FunctionComponent<AgencyIconProps> = ({ agency, classNa
     "phcorp": PHCorpIcon,
     "spr": SPRIcon,
     "unhcr": UNHCRIcon,
+    "bpan": MOTIcon,
+    "bppib": BOMBAIcon,
```

Comment:
Not an agency. Should not be placed here. Else, in the future, we have to start including all division level unit inside. 

Requires a rework of the AgencyBadge component to make it flexible (again). @leneherng pls take note, since you refactored the component before